subsys/bluetooth/host/gatt.c

```diff
@@ -2252,6 +2252,14 @@ static void gatt_find_info_rsp(struct bt_conn *conn, u8_t err,
 			continue;
 		}
 
+		/* Skip attributes that are not considered descriptors */
+		if (!bt_uuid_cmp(&u.uuid, BT_UUID_GATT_PRIMARY) ||
```

Comment:
I assume there's no direct use of `Find Information Request/Response` by the application.
But what about characteristic values? those will still come through won't they? does this make sense?

Reply:
Well we can't possible filter all of the characteristics, that table would be too big besides that would have to include vendor one as well so I don't think it is doable, we could perhaps add some state tracking for the declaration and then skip the next handle.

Reply:
Right but is it worth it? Shouldn't the user know to pass the correct range when discovering descriptors?